GroupsFromLDAP.groovy

```diff
@@ -0,0 +1,71 @@
+import java.util.Hashtable
+import javax.naming.Context
+import javax.naming.NamingEnumeration
+import javax.naming.NamingException
+import javax.naming.directory.*
+import javax.naming.ldap.*
+import jenkins.model.Jenkins;
+import nectar.plugins.rbac.strategy.*;
+import hudson.security.*;
+import nectar.plugins.rbac.groups.*;
+import nectar.plugins.rbac.roles.*;
+
+
+try {
+        String ldapAdServer = "ldap://192.0.2.36:389"
```

Comment:
Just a suggestion "ldap://192.0.2.36:389" could be changed for ldap://ldap.yourcompany.com:389 to keep consistency with https://wiki.jenkins-ci.org/display/JENKINS/LDAP+Plugin